databricks/koalas/generic.py

```diff
@@ -1393,8 +1393,9 @@ def median(self, accuracy=10000):
         # This is expected to be small so it's fine to transpose.
         return DataFrame(sdf)._to_internal_pandas().transpose().iloc[:, 0]
 
-    def rolling(self, *args, **kwargs):
-        return Rolling(self)
+    # TODO: 'center', 'win_type', 'on', 'axis' parameter should be implemented.
+    def rolling(self, window, min_periods=None):
```

Comment:
@itholic can we add a doc here or in https://github.com/databricks/koalas/pull/990?

Reply:
okay i'll add :)